router_client.go

```diff
@@ -284,6 +284,13 @@ type SendPaymentRequest struct {
 
 	// AMP is set to true if the payment should be an AMP payment.
 	AMP bool
+
+	//  If set, the payment loop can be interrupted by manually canceling
```

Comment:
nit: Godoc comments should start with the field name. Also, there are double spaces at the start of the comment.